test/cli/test_cli_utils.py

```diff
@@ -104,32 +105,65 @@ def test_add_common_arguments():
 
     options = parser.parse_args([])
     assert hasattr(options, 'config')
-    assert options.config is None
+    assert hasattr(options, 'configdir')
+    assert options.config == 'default'
+    assert options.configdir == config.DEFAULT_HOMEDIR
 
     options = parser.parse_args(['-c', 'test-short'])
     assert options.config == 'test-short'
 
     options = parser.parse_args(['--config', 'test-long'])
     assert options.config == 'test-long'
 
+    options = parser.parse_args(['--config-dir', 'test-long'])
+    assert options.configdir == 'test-long'
+
+    options = parser.parse_args(
+        ['-c', 'test-short', '--config-dir', 'test-long-dir'])
+    assert options.config == 'test-short'
+    assert options.configdir == 'test-long-dir'
+
+    options = parser.parse_args(
+        ['--config', 'test-long', '--config-dir', 'test-long-dir'])
+    assert options.config == 'test-long'
+    assert options.configdir == 'test-long-dir'
+
 
 def test_add_common_arguments_subparser():
-    """Assert function adds the -c/--config option on a subparser."""
+    """Assert function adds the multiple options on a subparser.
+
+    The expected options are -c/--config and --config-dir.
```

Comment:
Backticks for option names?

Reply:
In a test, unless we want to generate a doc from them (we could, that can have values), I'm not really for it.

Reply:
I think it's safer to write docstrings in the same style no matter where they appear, whether that file is included in the generated docs or not _at present_, so we don't have to chase down bad formatting later if/when the docs start pulling in that file.